lib/contestant.rb

```diff
@@ -0,0 +1,24 @@
+class Contestant
+  attr_reader :age, :state_of_residence, :spending_money, :game_interests
+  def initialize(argument)
```

Comment:
Nice job only creating the attrs that you needed and deconstructing the hash argument. The naming for argument could be improved though ;)